### DISCUSSION TITLE
Why are my images rendering as empty space?

### DISCUSSION THREAD

**Discussion by StartAutomating** [community/community#37413] [Other Feature Feedback, Questions, & Ideas]
This one continues to vex me.

I've got a number of sites with existing assets.

SVG assets seem to render fine, but everything else seems to break in interesting ways:

.GIFs render as a transparency of the correct dimensions
.PNGs render as a broken image link

One of my repositories with this issue is [EZOut](https://github.com/StartAutomating/EZOut).  

[It's current GitHub Page](https://ezout.start-automating.com) will show copious blank space where images should be.

I recently built a little "[preview page](https://ezout.start-automating.com/ListAssets)" for static files to see if that would help me narrow down the problem.  [Here is that page's source](https://github.com/StartAutomating/EZOut/blob/master/docs/ListAssets.md).  No dice.

I've also asked variations of this theme on [StackOverflow](https://stackoverflow.com/questions/74090445/why-arent-jekyll-assets-displaying-correctly), to no avail.

What the heck is going on here?

**Comment by ossanna16** [comment]
Hi there @StartAutomating and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by StartAutomating** [comment]
In case anyone else faces similar problems, in my case, it was because a tool I was using in a workflow was mangling the copy.

Long story made short: the file was re-encoded as text, which broke .gifs in different ways then .pngs and .svgs.

* .gifs were transparent but present
* .pngs were broken
* .svgs were fine (because .svg is just xml)

See the [stackoverflow](https://stackoverflow.com/questions/74090445/why-arent-jekyll-assets-displaying-correctly) question for details.
